wdl/src/main/scala/wdl/WdlNamespace.scala

```diff
@@ -102,40 +101,6 @@ case class WdlNamespaceWithWorkflow(importedAs: Option[String],
 
   override def toString: String = s"[WdlNamespace importedAs=$importedAs]"
 
-  /**
-    * Confirm all required inputs are present and attempt to coerce raw inputs to `WomValue`s.
-    * This can fail if required raw inputs are missing or if the values for a specified raw input
-    * cannot be coerced to the target type of the input as specified in the namespace.
-    */
-  def coerceRawInputs(rawInputs: ExecutableInputMap): Try[WorkflowCoercedInputs] = {
```

Comment:
Note to reviewers: In case you're curious why this is going, it was unused apart from in tests.

Reply:
and who needs tests?

Reply:
To be clear: this function was only ever used by tests that only tested this function :)